src/CSET/recipes/generic_surface_spatial_plot_sequence_case_aggregation_mean_hour_of_day.yaml

```
@@ -1,7 +1,7 @@
 category: Surface Spatial Plot
 title: "$MODEL_NAME Surface $VARNAME \n Aggregation by hour of day"
```

```suggestion
title: "$MODEL_NAME Surface $VARNAME\n Aggregation by hour of day"
```
I know this line predates this PR, but the same thing applies.